compiler/scenario-service/client/src/DA/Daml/LF/ScenarioServiceClient.hs

```diff
@@ -118,7 +117,7 @@ data Context = Context
   { ctxModules :: MS.Map Hash (LF.ModuleName, BS.ByteString)
   , ctxPackages :: [(LF.PackageId, BS.ByteString)]
   , ctxDamlLfVersion :: LF.Version
-  , ctxLightValidation :: LowLevel.LightValidation
+  , ctxValidation :: Bool
```

Comment:
same here, this should also be a newtype (you can use the same newtype for both if you want).